site/src/components/LicenseCard/LicenseCard.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { screen } from "@testing-library/react"
+import { render } from "../../testHelpers/renderHelpers"
+import { LicenseCard } from "./LicenseCard"
+import { MockLicenseResponse } from "testHelpers/entities"
+
+describe("LicenseCard", () => {
+  it("renders (smoke test)", async () => {
+    // When
+    render(
+      <LicenseCard
+        license={MockLicenseResponse[0]}
+        userLimitActual={1}
+        userLimitLimit={10}
+        onRemove={() => null}
+        isRemoving={false}
+      />,
+    )
+
+    // Then
+    await screen.findByText("#1")
+    await screen.findByText("1 / 10")
+    await screen.findByText("Enterprise")
```

Comment:
It doesn't make sense to look at the entire screen for `#1` as I'm sure that you will find something else. Same for Enterprise. Could we limit it only to the rendered element?

Reply:
I think this is ok and a common practice for JS tests when using react testing library

Reply:
The practice is fine, but we should limit it to the element under tests.

Reply:
I mean it is safe because if the matcher finds it in multiple elements, it will raise an error.

Reply:
What if it finds a static label placed somewhere else on the page, and our license card didn't render at all?

Reply:
The storybook should get that. IMO, integration tests should be used to catch network and "actionable" things. Check if things are rendered correctly should be on Storybook but I get your point.

Reply:
it is usually a recommendation to use `screen`: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

And also in this case we are only rendering the component, there isn't anything else in the screen that could be matched. I have printed the result of a `screen.debug()` and this is the result: 

```      
<body>
        <div>
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation2 makeStyles-licenseCard-27 css-19zg6r2-MuiPaper-root"
          >
            <div
              class="makeStyles-stack-39 makeStyles-stack-40  makeStyles-cardContent-28"
            >
              <span
                class="makeStyles-licenseId-29"
              >
                #
                1
              </span>
              <span
                class="makeStyles-accountType-30"
              >
                Enterprise
              </span>
              <div
                class="makeStyles-stack-39 makeStyles-stack-41"
                style="flex: 1;"
              >
                <div
                  class="makeStyles-stack-39 makeStyles-stack-42"
                >
                  <span
                    class="makeStyles-secondaryMaincolor-33"
                  >
                    Users
                  </span>
                  <span
                    class="makeStyles-userLimit-26"
                  >
                    1

                     / Unlimited
                  </span>
                </div>
                <div
                  class="makeStyles-stack-39 makeStyles-stack-43"
                  width="134px"
                >
                  <span
                    class="makeStyles-secondaryMaincolor-33"
                  >
                    Valid Until
                  </span>
                  <span
                    class="makeStyles-licenseExpires-31"
                  >
                    May 20, 2078
                  </span>
                </div>
                <div
                  class="makeStyles-stack-39 makeStyles-stack-44"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textNeutral MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textNeutral MuiButton-sizeSmall MuiButton-textSizeSmall makeStyles-removeButton-34 css-1sr08q4-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    Remove
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </body>
```

as you can see, just the component is being rendered.